<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>154</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel{
	font: bold 10pt &quot;Open Sans&quot;;
	color: #353942;
}

QPushButton{
	font: 8pt &quot;Open Sans&quot;;
	color: #353942;
}

QComboBox{
	font: 8pt &quot;Open Sans&quot;;
	color: #353942;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0" colspan="2">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_camera">
        <property name="text">
         <string>Choose Camera Address:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="line_camera"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_gate">
        <property name="text">
         <string>Choose Gate Address:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="line_gate"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_port">
        <property name="text">
         <string>Choose Port Address:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="line_port"/>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="button_enter">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Enter</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QComboBox" name="comboBox">
      <property name="minimumSize">
       <size>
        <width>209</width>
        <height>30</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>Gate 1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Gate 2</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
